### PR TITLE
Update turn-on-or-off-skype-for-business-client-feedback-reporting.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-skype-for-business-client-feedback-reporting.md [hunk 1]
```diff
@@ -37,39 +37,23 @@ You can also use the  _EnableOnlineFeedbackScreenshot_ setting so users can incl
 > [!IMPORTANT]
 > The logs collected by the app's feedback tool will be stored for up to a maximum of 90 days in the United States while the issue is being investigated. Because of this, please don't enable this feedback tool if this violates your organization's data protection policy. 
   
-## Verify and start Windows PowerShell
+## Start Windows PowerShell
 
-- **Check that you are running Windows PowerShell version 3.0 or higher**
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-skype-for-business-client-feedback-reporting.md:42]:
```suggestion
> [!NOTE]
```

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-skype-for-business-client-feedback-reporting.md [hunk 2]
```diff
@@ -37,39 +37,23 @@ You can also use the  _EnableOnlineFeedbackScreenshot_ setting so users can incl
...
+  > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-skype-for-business-client-feedback-reporting.md:43]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**